### PR TITLE
Bump version, prepare empty SQL file and migration script

### PR DESCRIPTION
**Pull request by trasher**
Also a few very minor fixes

### CODE REVIEW

File: install/update.php [hunk 1]
```diff
@@ -795,7 +795,11 @@ function updateDbUpTo031() {
          update0905to91();
 
       case "9.1" :
-      case "0.91": // // for change name of the version - to delete in next version
+         include_once("update_91_92.php");
```
Comment (remicollet) [inline install/update.php]: Perhaps. better to keep 0.91, even if never released

File: install/update_xx_xy.tpl.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+/** @file
+* @brief
+*/
+
+/**
+ * Update from 9.1 to 9.2
+ *
+ * @return bool for success (will die for most error)
+**/
+function update91to92() {
```
Comment (orthagh) [inline install/update_xx_xy.tpl.php]: for template, maybe we could replace defined version by some tags (xx & xy) .

Comment (trasher) [inline install/update_xx_xy.tpl.php]: You're right, will do.

Comment (trasher) [inline install/update_xx_xy.tpl.php]: Done

**Pull request merged**